src/qml/ChatPage.qml

```diff
@@ -59,18 +59,23 @@ Kirigami.Page {
 				Rectangle {
 					id: avatar
 
-					width: height
-					height: parent.height
+					width: 40
+					height: 40
+					radius: width*0.5
 					color: "grey"
 					visible: !sentByMe
 				}
 
 				Rectangle {
-					width: 80
-					height: 40
+					radius: 2
```

Comment:
... request what you want (even if it's money :laughing:) - we have a comment function for discussing :smiley_cat: